python/eos/plot/plotter.py

```diff
@@ -461,10 +461,21 @@ def plot(self):
             if 'parameters' in item and 'parameters-from-file' in item:
                 eos.warn('    overriding values read from \'parameters-from-file\' with explicit values in \'parameters\'')
 
+            if 'parameters' in item and 'parameters-from-mode' in item:
+                eos.warn('    overriding values read from \'parameters-from-mode\' with explicit values in \'parameters\'')
+
             if 'parameters-from-file' in item and type(item['parameters-from-file']) is str:
                 eos.warn('    overriding parameters from file')
                 parameters.override_from_file(item['parameters-from-file'])
 
+            if 'parameters-from-mode' in item and type(item['parameters-from-mode']) is str:
+                eos.warn('    overriding parameters from mode')
+                mode = eos.Mode(item['parameters-from-mode'])
+                if not mode.type == "Mode":
```

Comment:
Good to check here. But the check should be part of the data file class, really. If it is not yet, let's open an issue.

Reply:
It is already :)

Reply:
Then this can go, right? I would prefer removing it. Suppose we switch from a ``Mode`` to ``Mode-v2`` format in the future, this might lead to unexpected bugs.